### DISCUSSION TITLE
实现多级 list 页面的功能(面包屑导航)

### DISCUSSION THREAD

**Discussion by boh5** [hugo-fixit/FixIt#180] [Ideas]
假如我有如下目录：
```
$ tree content
.
└── content/
    └── notes/
        ├── languages/
        │   ├── python/
        │   │   ├── 1.md
        │   │   └── 2.md
        │   └── golang/
        │       ├── 3.md
        │       └── 4.md
        └── databases/
            ├── mysql/
            │   ├── 5.md
            │   └── 6.md
            └── redis/
                ├── 7.md
                └── 8.md
```
- 访问 `/notes`，返回 `notes` 的导航页面，`luanguages` 和 `databases` 的链接
- 访问 `/notes/luanguages`，返回 `languages` 的导航页面，`python` 和 `golang` 的链接
- 访问 `/notes/luanguages/python`，返回 `python` 文章的列表，`1.md` 和 `2.md` 的链接

我尝试用 hugo 的 `Branch Bundle` 来做，但似乎实现不了。在深层目录中新建 `_index.md` 后，上层 section 就感受不到下层的文章了，不知道是不是我搞错了，例如：

- 上面的目录结构下，访问 `/notes` 会返回所有文章
- 新建 `notes/languages/_index.md`
- 再访问 `/notes`，就没有 `languages` 下的文章了
- 要访问 `/notes/languages`，才能看到 `languages` 下的文章

请教该怎么实现，或者在 `FixIt` 中实现该功能？谢谢🙏

**Comment by Lruihao** [comment]
面包屑导航已支持 #309
